src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import { ToDo } from '../../types/ToDo';
+import cs from 'classnames';
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { findUserById } from '../../service/UserService';
+
+type Props = {
+  todo: ToDo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cs('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={findUserById(todo.userId)} />
```

Comment:
It's better to pass a todo with a user info directly to the `TodoInfo` component instead of doing it there.
This component should be just a presentational("dummy") component, accept props and render UI, that's all